### ISSUE TITLE
Fix typo in documentation

### ISSUE THREAD

**Issue by TerensTare** [erincatto/box2d#676]
I was reading the docs today and came across a small typo on the `for` loop right before this section:
https://box2d.org/documentation/md__d_1__git_hub_box2d_docs_collision.html#autotoc_md41
```cpp
// Visit each child edge.
for (int32 i = 0; i \< chain.GetChildCount(); ++i) // notice the \<, probably should be <
{
    b2EdgeShape edge;
    chain.GetChildEdge(&edge, i);
 
    ...
}
```
I encountered the same typo in the first example of [Contact Manifolds](https://box2d.org/documentation/md__d_1__git_hub_box2d_docs_collision.html#autotoc_md46)
```cpp
b2WorldManifold worldManifold;
worldManifold.Initialize(&manifold, transformA, shapeA.m_radius,
transformB, shapeB.m_radius);
 
for (int32 i = 0; i \< manifold.pointCount; ++i) // \<
{
    b2Vec2 point = worldManifold.points[i];
    ...
}
```

**Comment by HughPH** [comment]
Backslash (or "Backtick") is used to escape characters that would (or could) otherwise be misinterpreted. In these cases, it's reasonable to expect that < might be misinterpreted as the beginning of a HTML tag in the final output, and so it's been escaped.

https://www.markdownguide.org/basic-syntax/

**Comment by TerensTare** [comment]
I am not 100% sure, but the characters inside a  code block (triple "backtick") should be matched according to the coding language specified. Here is the second code snippet I posted earlier but without the backslash:
```cpp
b2WorldManifold worldManifold;
worldManifold.Initialize(&manifold, transformA, shapeA.m_radius,
transformB, shapeB.m_radius);
 
for (int32 i = 0; i < manifold.pointCount; ++i)
{
    b2Vec2 point = worldManifold.points[i];
    ...
}
```
Once again: I have no idea how doxygen will interpret this. Thank you for the heads up anyway.

**Comment by erincatto** [comment]
Fixed these, but then found out latest doxygen breaks my image format.

https://github.com/doxygen/doxygen/issues/8462

Looking for a fix.

**Comment by erincatto** [comment]
Fix here 0535dd1f64b8c1f0448d025bb284eb5cb67bb9b9

Still not happy with images. Lost captions. Hopefully doxygen will fix this.
